### PR TITLE
Ditac config file

### PR DESCRIPTION
**Pull request by ditac**
*Description of changes:*

Add the ability to specify the properties file via a parameter during startup.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java [hunk 1]
```diff
@@ -60,13 +66,13 @@ public String getSettingValue(String settingName, String defaultValue) {
...
         if (httpsEnabledString == null) {
-            instance.httpsEnabled = false;
+            httpsEnabled = false;
         }
         try {
-            instance.httpsEnabled = Boolean.parseBoolean(httpsEnabledString);
+            httpsEnabled = Boolean.parseBoolean(httpsEnabledString);
         } catch (Exception ex) {
             LOG.error("Unable to parse httpsEnabled property with value {}", httpsEnabledString);
             instance.httpsEnabled = false;
```
Comment (yeajung) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java]: should this be "httpsEnabled = false" as well?

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java [hunk 2]
```diff
@@ -77,58 +83,59 @@ public boolean getHttpsEnabled() {
         return this.httpsEnabled;
```
Comment (yeajung) [inline src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java]: no "this"?

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java (hunk 1 above)
Comment (ditac) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java]: Fixed.

File: src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java (hunk 2 above)
Comment (ditac) [reply src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/config/PluginSettings.java]: I think going forward we should use this, for all member variables to make it easier to read.

**Pull request merged**